python/cdp-agentkit-core/cdp_agentkit_core/actions/get_balance.py

```diff
@@ -1,12 +1,13 @@
 from collections.abc import Callable
+from typing import Optional
 
-from cdp import Wallet
+from cdp import Cdp, Wallet
 from pydantic import BaseModel, Field
 
 from cdp_agentkit_core.actions import CdpAction
 
 GET_BALANCE_PROMPT = """
-This tool will get the balance of all the addresses in the wallet for a given asset.
+This tool will get the balance of address.
```

Comment:
Please see suggestion
```suggestion
This tool will get the balance of all the addresses in the wallet for a given asset, or for a user-specified address if provided.
```